README.md

```diff
@@ -870,10 +870,14 @@ ANSWERS.addComponent('Pagination', {
   container: '.pagination-component',
   // Required*, the vertical for pagination, *if omitted, will fall back to the search base config
   verticalKey: 'verticalKey',
-  // Optional, display a double arrow allowing users to jump to the first page of results
```

Comment:
When we deprecate an option but it is still used in our code, do we want to mark it as deprecated instead of removing it entirely until it is phased out? I can see this being confusing when an HH is using it, upgrades, and finds that it is no longer in the README but it is still working. Unsure if this is something we want to support

Reply:
asking Tom about his thoughts!

Reply:
Tom said that what we've been doing is just removing any deprecated options from the readme. After bringing up your points I think the verdict is we should keep these around and mark them as deprecated
> Hmm, the Theme could reference older, deprecated SDK stuff. We'd want to keep the Theme up-to-date as much as possible, but that's no guarantee. Maybe we should still refer to it in the README (but note that it's deprecated and they should use [the new option])

Reply:
updated!